src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -363,9 +363,19 @@ export class NotebookModel extends Disposable implements INotebookModel {
 						}
 					});
 				}
+				// Get Notebook URI Params & adjust to string
+				let notebookUriParams: string = this.notebookUri?.query;
+				notebookUriParams = notebookUriParams.split('').join(' ').replace('& ', '\n');
+				// Get parameter cell and index to place new notebookUri parameters accordingly
+				let parameterCellIndex = 0;
+				let parameterCell = false;
```

Comment:
hasParameterCell might be a better variable name

Reply:
Yeah, that would make it clear this notebook _has_ a parameter cell.